bot/seasons/easter/egg_facts.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class EasterFacts(commands.Cog):
+    """A cog for easter egg facts."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.facts = self.load_json()
+
+    @staticmethod
+    def load_json():
+        """Loading the json data"""
+        p = Path('bot', 'resources', 'easter', 'easter_egg_facts.json')
+        with p.open(encoding="utf8") as f:
+            facts = load(f)
+        return facts
+
+    async def background(self):
+        """A background task that sends a easter egg fact."""
+        channel = self.bot.get_channel(426566445124812815)
```

Comment:
I believe this is a leftover from your tests, isn't it? It should use one of the constants instead unless I'm mistaking it

Reply:
its the seasonalbot channel id ,i will change it once PR #198 is merged as it introduces the channel id in the constants file.

Reply:
#198 has been merged so this can be cleaned up.